api/v1alpha1/connection_types.go

```diff
@@ -17,6 +17,7 @@ type Connection struct {
 	// +optional
 	Limit *ConnectionLimit `json:"limit,omitempty"`
```

Comment:
Now we get two Limits in this struct. :-)  Should we change this to `ConnectionLimit`?

@guydc @arkodg

Reply:
sure yah, not ideal, but lets do it